Gemfile

```diff
@@ -3,8 +3,11 @@ ruby '2.4.2'
 
 gem 'activesupport'
 gem 'flydays', git: 'https://github.com/kevinreedy/flydays.git', ref: '2e8d88f446bd5257c3e6da865d656f1f697dc4a0'
+gem 'rake'
 gem 'sinatra'
 gem 'shotgun'
+gem 'vcr'
+gem 'webmock'
```

Comment:
I'd move these into a:

```
group :test do
  gem 'vcr'
  gem 'webmock'
end
```

... so the fakery gems aren't needed in production.

Reply:
Er ... or, I seem to be misunderstanding their use. Imma read more code.

Reply:
Ooooh. Uh. I see what you're doing here.

I'm not sure how I feel about it.

![tenor-246286052](https://user-images.githubusercontent.com/517302/34062646-b8cbddb2-e1bb-11e7-8a7a-82efd794f7cd.gif)